RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterGPUKernel.cu

```diff
@@ -509,18 +510,16 @@ namespace pixelgpudetails {
     }
 
 #ifdef GPU_DEBUG
-    uint16_t maxH = isPhase2 ? 3027 : 1024;
+    uint16_t maxH = (isPhase2) ? 3027 : 1024;
```

Comment:
I'd leave the parentheses out, as they are clearly not needed, and in some cases I've seen a warning about unnecessary parenthesis